src/sql/magic.py

```diff
@@ -180,6 +181,26 @@ def _mutex_autopandas_autopolars(self, change):
                 setattr(self, other, False)
                 print(f"Disabled '{other}' since '{change['name']}' was enabled.")
 
+    def check_random_arguments(self, line="", cell=""):
```

Comment:
still mixing changes from two PRs. please verify that changes of your PR before requesting review since a few of your PRs have the same problem

Reply:
Got it! I will be careful submitting PR in the next time. I have fixed and updated it now.